compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -436,12 +436,14 @@ struct __sanitizer_tm {
   int tm_wday;
   int tm_yday;
   int tm_isdst;
-#if SANITIZER_HAIKU
+#  if !SANITIZER_AIX
+#    if SANITIZER_HAIKU
   int tm_gmtoff;
 #else
   long int tm_gmtoff;
 #endif
```

Comment:
Formatting inconsistency?

Reply:
@jakeegan, I think this (and similar other cases) points to a need to land an NFC auto-format patch on more parts of the code first.